packages/amplify-e2e-tests/src/__tests__/rds-mysql-v2-generate-schema.test.ts

```diff
@@ -0,0 +1,16 @@
+import { ImportedRDSType } from '@aws-amplify/graphql-transformer-core';
+import { testRDSGenerateSchema } from '../rds-v2-tests-common/rds-v2-generate-schema';
+
+// to deal with bug in cognito-identity-js
+(global as any).fetch = require('node-fetch');
```

Comment:
Do we need this even though we're not doing any auth in the test?

Reply:
Will remove this and see in subsequent PR.